clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -124,6 +125,9 @@ private void handleMetalakeCommand() {
 
     if (CommandActions.DETAILS.equals(command)) {
       new MetalakeDetails(url, ignore, metalake).handle();
+      if (line.hasOption(GravitinoOptions.AUDIT)) {
+        new MetalakeAuditInfo(url, ignore, metalake).handle();
+      }
```

Comment:
It might be best to make it display audit info or details or not both at once.

Reply:
![image](https://github.com/user-attachments/assets/2671132e-9668-41a3-bda6-3e0b13bd62cc)

fixed